nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -319,6 +320,15 @@ public class ConsumeKinesisStream extends AbstractAwsAsyncProcessor<KinesisAsync
             .dependsOn(RECORD_WRITER)
             .build();
 
+    public static final PropertyDescriptor FLOW_FILE_HANDLING_ON_SCHEMA_CHANGE_STRATEGY = new PropertyDescriptor.Builder()
+            .name("FlowFile Handling On Schema Change Strategy")
```

Comment:
The name can be adjusted along the lines of the enum.
```suggestion
            .name("FlowFile Handling On Schema Difference")
```